### PR TITLE
Add support for compiling HTMLBars

### PR DESCRIPTION
**Pull request by kenips**

### CODE REVIEW

**Comment by rwjblue** [comment]
Can you rebase (just merged https://github.com/rwjblue/broccoli-ember-inline-template-compiler/pull/7)?

**Comment by kenips** [comment]
yep. done.

File: index.js [hunk 1]
```diff
@@ -2,20 +2,23 @@
...
 function EmberInlineTemplatePrecompiler (inputTree, options) {
-  if (!(this instanceof EmberInlineTemplatePrecompiler)) return new EmberInlineTemplatePrecompiler(inputTree, options);
+  if (!(this instanceof EmberInlineTemplatePrecompiler)) {
+    return new EmberInlineTemplatePrecompiler(inputTree, options);
+  }
 
+  this.options = options || {};
   this.inputTree = inputTree;
-  this.compiler = options && options.compiler || require('ember-template-compiler');
+  this.compiler = options && options.compiler || this.options.templateCompiler;
```
Comment (kenips) [inline index.js]: actually want to use options.compiler for everything?

File: package.json [hunk 2]
```diff
@@ -18,9 +18,8 @@
     "javascript"
   ],
   "dependencies": {
-    "broccoli": "0.13.0",
-    "broccoli-filter": "0.1.7",
-    "ember-cli-version-checker": "^1.0.2",
-    "ember-template-compiler": "1.9.0-alpha"
+    "broccoli": "0.15.0",
```
Comment (rwjblue) [inline package.json]: This should be a development dependency only...

File: index.js (hunk 1 above)
Comment (rwjblue) [inline index.js]: Yes, that sounds good.

**Comment by kenips** [comment]
updated!

**Pull request closed without merging**